### PR TITLE
[Issue-7] PageFooter and PageHeader generation

### PR DESCRIPTION
**Pull request by ScriptHound**
added generation of a page_header and page_footer for text

### CODE REVIEW

File: lib/src/text.rs [hunk 1]
```diff
@@ -219,10 +219,35 @@ impl TransformerTrait for Transformer {
             )?;
         }
 
+        for element in &document.page_header {
```
Comment (evgenyigumnov) [inline lib/src/text.rs:222]: покройте тестами свое решение

**Review by evgenyigumnov — CHANGES_REQUESTED** [review]
resolve my comments please

**Comment by ScriptHound** [comment]
Tests were added but rather visual. And im still hesitating what PageFooter and PageHeader should look like but now the whole document looks like

`my header string

First header 
  
 1. List item 1 
 2. List item 2 
 3. List item 3 
  
 Paragraph  bla bla bla blabla bla bla blabla bla bla blabla bla bla blabla bla bla blabla bla bla blabla bla bla blabla bla bla blabla bla bla 
 blabla bla bla blabla bla bla blabla bla bla blabla bla bla bla 
  
 Second header 
  
 +-----------------+-----------------+ 
 | Header 1        | Header 2        | 
 +-----------------+-----------------+ 
 | Row 1, Column 1 | Row 1, Column 2 | 
 | Row 2, Column 1 | Row 2, Column 2 | 
 +-----------------+-----------------+ 
 

my footer string
`

**Review by evgenyigumnov — APPROVED** [review]
Ok, thanks

**Pull request merged**